packages/shared_preferences/lib/shared_preferences.dart

```diff
@@ -45,8 +45,8 @@ class SharedPreferences {
   /// in sync since the setter method might fail for any reason.
   final Map<String, Object> _preferenceCache;
 
-  /// returns the entire set for debugging purposes.
-  Map<String, Object> getAll() => _preferenceCache;
+  /// Returns all keys in the preferenceCache
```

Comment:
[nit]. The Dart [style guide](https://www.dartlang.org/guides/language/effective-dart/documentation#do-make-the-first-sentence-a-standalone-paragraph) calls for a `.` to end the first paragraph of dartdoc.

Reply:
This is how you keep the codebase clean 👍 
Added the dot and did another push!